parts/linux/cloud-init/artifacts/mariner/mariner-package-update.sh

```diff
@@ -48,6 +48,49 @@ if [ -n "${current_timestamp}" ]; then
     fi
 fi
 
+# Network isolated cluster can't access the internet, so we deploy a live patching repo service in the cluster
+# The node will use the live patching repo service to download the repo metadata and packages
+# If the annotation is not set, we will use the ubuntu snapshot repo
+live_patching_repo_service=$($KUBECTL get node ${node_name} -o jsonpath="{.metadata.annotations['kubernetes\.azure\.com/live-patching-repo-service']}")
```

Comment:
should we log something out of kubectl fails? you can grab the exit code with `$?`

Reply:
Since we added `set -e` for this script, if kubectl fails, it will exit directly and also log the reason.